examples/inventory.lua

```diff
@@ -0,0 +1,42 @@
+meta.name = "Inventory Example"
+meta.version = "0.1"
+meta.description = "Examples for using player inventory."
+meta.author = "jeremyhay"
+
+local names = {}
+for i,v in pairs(ENT_TYPE) do
+  names[v] = i
+end
+
+-- display the items and the powerups in the player's inventory
+set_callback(function()
+    if players and players[1] then
+        x, y, l = get_position(players[1].uid)
+        -- you may want to mask only the items and logical (powerups)
+        -- here so you don't get any unwanted FX
+        -- note that ITEM_WHIP shows up while you are whipping
+        local ANY_ITEM = 0
+        local items = entity_get_items_by(players[1].uid, ANY_ITEM, MASK.ITEM & MASK.LOGICAL)
```

Comment:
What's `MASK.ITEM & MASK.LOGICAL`? If you want those two bits then you should do `MASK.ITEM | MASK.LOGICAL`

Reply:
you're right I don't know what I was thinking. Been a while since I used bitwise operators.